### DISCUSSION TITLE
تمام اینباندها در یک فایل

### DISCUSSION THREAD

**Discussion by s43id** [Gozargah/Marzban#506] [Show and tell]
**در این فایل زیپ میتونید به اینباند های زیر دسترسی داشته باشید:**

### **Vless:**
Reality Tcp
Reality Grpc
Reality H2
Tcp+Tls
Tcp+NoTls
Tcp+Header
Ws+Tls
Ws+NoTls
Ws+Header
GRPC+Tls
GRPC+NoTls
.
.
### **Vmess:**
Tcp+Tls
Tcp+NoTls
Tcp+Header
Ws+Tls
Ws+NoTls
Ws+Header
GRPC+Tls
GRPC+NoTls
.
.
### **Trojan:**
Tcp+Tls
Tcp+NoTls
Tcp+Header
Ws+Tls
Ws+NoTls
Ws+Header
GRPC+Tls
GRPC+NoTls
.
.

**Shadowsocks:**
TCP,UDP

**[All Inbound.zip](https://github.com/Gozargah/Marzban/files/14548039/All.Inbound.zip)**

--------------------------------------------------

برای اینباند های ریلیتی پرایوت کی و پابلیک کی خودتون رو با استفاده از دستور:

**docker exec marzban-marzban-1 xray x25519**

بگیرید و مقادیر رو جایگزین کنید.

--------------------------------------------------

برای اینباند های **Tls** دار در بخش **Server_Name** دامنه خودتون رو وارد کنید و مسیر سرتیفیکت خودتون رو بهش بدید

**Comment by YasharMir** [comment]
دستت درد نکنه اینا همشون کار می کنن ؟ کدوما را میشه پشت cdn انداخت ؟

**Reply by s43id** [reply]
سلام بسته به نیازتون باید اینباند مد نظر رو بردارید و از وب سوکت و GRPC برای کانکشن پشت CDN میشه استفاده کرد البته grpc رو جاهایی مثل اروان کلود پشتیبانی نمیکنن اما وب سوکت همه جا اوکیه

**Comment by Saeedmhm** [comment]
ارور میده
Expecting value: line 2 column 17 (char 17)

**Reply by s43id** [reply]
از کدوم اینباند استفاده کردید که ارور داد؟

**Comment by y8nes** [comment]
سلام خط 511 بجای trojan نوشتید vmess

**Reply by Saeedmhm** [reply]
من خط دوم فقط ارور میگیرم
Expecting value: line 2 column 17 (char 17)

**Reply by s43id** [reply]
سلام مرسی که گفتی درستش کردم

**Comment by iamtheted** [comment]
سلام ، مرسی از فایل کامل شما. 
حضور شما در کامیونیتی مرزبان بسیار ارزشمنده. 💙

**Comment by DeAref** [comment]
دوستان نظرتونه این فایل رو آپدیت کنیم؟

**Reply by s43id** [reply]
شد

**Comment by Ten10vpn** [comment]
ممنون میشوم اینباند httpupgrade tls را هم قرار دهید

**Reply by iamtheted** [reply]
https://gozargah.github.io/marzban/fa/docs/xray-inbounds

**Comment by Eslender73** [comment]
رمزنگاری شادوساکس رو نمیشه اضافه کرد تو اینباند ؟ منظورم همون متوده
